src/test/java/com/uipath/uipathpackage/UiPathTestTests.java

```diff
@@ -67,7 +66,7 @@ public static void setupClass() {
         password = System.getenv("TestOrchestratorPassword");
         token = "testtoken";
         description = "TestDesc";
-        folderName = "JenkinsTests";
+        folderName = "MobileAutomationTesting";
```

Comment:
Let's use `System.getenv("TestOrchestratorFolderName")` instead. Make sure we pass it as an `env` in the pipeline, since it's also a secret.